### ISSUE TITLE
Jot Down detailed requirements for server.

### ISSUE THREAD

**Issue by ritikrishu** [UdacityMobileWebScholarship/showcase-app#30]
If we can get a detailed list of what else the server needs other than already covered in #29 . We can fasten up server setup process.

**Comment by palnabarun** [comment]
I will share a basic API design based on the wireframes and functionality already decided by tonight. So, I am assigning this to me also.

**Comment by ritikrishu** [comment]
yeah sounds good! @ everyone, please provide your input here so we pick/merge all things that we think is required.
